core/templates/dev/head/pages/collection-player-page/collection-player-page.directive.ts

```diff
@@ -250,6 +250,30 @@ oppia.directive('collectionPlayerPage', ['UrlInterpolationService',
               ctrl.collectionId);
           };
 
+          ctrl.getExplorationTitlePosition = function(index) {
+            if ( index % 2 === 0) {
+              return { width: '160',
+                position: 'absolute',
+                color: '#006553',
+                'font-size': '20px',
+                bottom: '70%', };
+            } else if ((index + 1) % 2 === 0 && (index + 1) % 4 !== 0) {
+              return { width: '160',
+                position: 'absolute',
+                color: '#006553',
+                bottom: '70%',
+                'font-size': '20px',
+                left: '20px' };
+            } else if ((index + 1) % 4 === 0) {
+              return { width: '160px',
+                position: 'absolute',
+                color: '#006553',
+                'font-size': '20px',
+                bottom: '70%',
+                left: '-35px' };
+            }
```

Comment:
Try to make it more generalized.
Like create one constant ``` exploration_title_css_dict ``` (doesn't know the naming convention).
Whose default value be
```
{
  width: '160px',
  position: 'absolute',
  color: '#006553',
  font-size: '20px',
}
```. And now add remaing key value pairs, according to if else condtion.

Reply:
@import-keshav hey buddy thanks for being so patient with me. I made the change you suggested and now I am begining to understand why opensource is encouraged so much. :D

Reply:
Thanks, @Showtim3
And welcome to opensource :)